libs/content/workspace-ffi/Cargo.toml

```diff
@@ -18,6 +18,7 @@ lb_external_interface = { path = "../../lb/lb_external_interface" }
 raw-window-handle = "0.6"
 jni = "0.21.0"
 pollster = "0.2"
+tracing="0.1.5"
```

Comment:
optional:
we should `pub use` tracing out of `lb` to avoid defining this multiple times